recipes/PayloadHttpApiNode/nodeSupported.md

```diff
@@ -0,0 +1,2 @@
+This file just indicates that the node backend is supported.
```

Comment:
This file should be named `nodeSupportedSkipCI.md`. Otherwise, CI will run this code and it will never terminate.